HackIllinois/APIService/HIEventService.swift

```diff
@@ -16,17 +16,17 @@ import CoreData
 
 final class HIEventService: HIBaseService {
     override static var baseURL: String {
-        return super.baseURL + "/event"
+        return super.baseURL + "event/"
     }
 
     // MARK: - Events
-    static func create(event: HIAPIEvent) -> APIRequest<HIAPIEvent.Contained> {
+    static func create(event: HIAPIEvent) -> APIRequest<HIAPIEvent> {
         let eventDict = [String: Any]()
-        return APIRequest<HIAPIEvent.Contained>(service: self, endpoint: "", body: eventDict, method: .POST)
+        return APIRequest<HIAPIEvent>(service: self, endpoint: "", body: eventDict, method: .POST)
     }
 
-    static func getAllEvents(active: Bool = false) -> APIRequest<HIAPIEvent.Contained> {
-        let paramaters = ["active": "\(active)"]
+    static func getAllEvents() -> APIRequest<HIAPIEvent.Contained> {
+        let paramaters = ["Content-Type": "application/json"]
```

Comment:
why?

Reply:
the active parameter isn't necessary as far as i know, should work as it was though

Reply:
no I mean `let paramaters = ["Content-Type": "application/json"]`

Reply:
not explicitly necessary, can be removed

Reply:
Just FYI, this data is never encoded as a `HTTPURLParameter`, it should be a `HTTPHeader`, however `HIBaseService` already does this for you.

Reply:
ok